### DISCUSSION TITLE
which version of cri-docker is adaptive for the version of k8s 1.28?

### DISCUSSION THREAD

**Discussion by wangyangsheng** [Mirantis/cri-dockerd#318] [General]
1 what 's the difference between pause3.6 and pause 3.9？
2 some errors as following appear when getting the tokens 

[root@master1 kubernetes]# kubeadm token create 3otfgv.avfzl45qs449g30u --v=5
I0213 19:32:19.513539    3981 token.go:119] [token] validating mixed arguments
I0213 19:32:19.513867    3981 token.go:128] [token] getting Clientsets from kubeconfig file
I0213 19:32:19.513886    3981 cmdutil.go:94] Using kubeconfig file: /etc/kubernetes/kubelet.conf
I0213 19:32:19.516522    3981 token.go:243] [token] loading configurations
I0213 19:32:19.517061    3981 interface.go:432] Looking for default routes with IPv4 addresses
I0213 19:32:19.517070    3981 interface.go:437] Default route transits interface "enp0s3"
I0213 19:32:19.517561    3981 interface.go:209] Interface enp0s3 is up
I0213 19:32:19.518280    3981 interface.go:257] Interface "enp0s3" has 3 addresses :[192.168.0.130/24 192.168.0.133/32 fe80::9fc9:38a3:51d4:2d09/64].
I0213 19:32:19.518304    3981 interface.go:224] Checking addr  192.168.0.130/24.
I0213 19:32:19.518313    3981 interface.go:231] IP found 192.168.0.130
I0213 19:32:19.518461    3981 interface.go:263] Found valid IPv4 address 192.168.0.130 for interface "enp0s3".
I0213 19:32:19.518468    3981 interface.go:443] Found active IP 192.168.0.130
I0213 19:32:19.518498    3981 kubelet.go:196] the value of KubeletConfiguration.cgroupDriver is empty; setting it to "systemd"
I0213 19:32:19.532104    3981 token.go:250] [token] creating token

**Comment by afbjorklund** [comment]
I think you can use the latest version of cri-dockerd, for any version of kubernetes (1.24-1.29) and for any docker 19.03 up

There is no major difference between pause:3.6 and pause:3.9 as far as I know - for the Linux images, only fixes for Windows.
